pipelines/batch/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # and run the docker image for you.  
 
 
-FROM adoptopenjdk/maven-openjdk11
+FROM eclipse-temurin:11-jdk-focal
```

Comment:
can we change to this in all the other Dockerfiles in the repo?

Reply:
This was actually causing a weird problem ([example](https://pantheon.corp.google.com/cloud-build/builds;region=global/c47bb87d-a9fa-4be1-a22e-81d0347f7796;step=17?project=cloud-build-fhir)), i.e., a DB connection issue that I had no explanation for. With trial and error I reverted some of the changes in this PR and finally reverting this, fixed the problem. For now I replace this with a TODO.